src/submission/exceptions/exceptions.ts

```diff
@@ -287,8 +291,11 @@ export async function getExceptionManifestRecords(
 	filters: { donorIds: number[]; submitterDonorIds: string[] },
 ): Promise<ExceptionManifestRecord[]> {
 	const { donorIds, submitterDonorIds: querySubmitterIds } = filters;
+	const filteredDonors = Boolean(donorIds.length || querySubmitterIds.length);
 
-	const donorsByDonorId = await getDonorsByIds(donorIds);
+	const donorsByDonorId = filteredDonors
+		? await getDonorsByIds(donorIds)
+		: (await getDonors(programId)).map((donor) => donor);
```

Comment:
If donorIds and submitterIds are empty, then we return all exceptions for all donors

Reply:
```ts
[...].map((x) => x)
```
will just return the original array, nothing is changing about the property you're iterating on